src/controllers/task.js

```diff
@@ -33,49 +12,31 @@ const TaskController = (taskModel) => {
             });
         }
         return res.status(200).json({
-            data: task
+            data: task,
+            error: err ? err.message : ""
         });
     });
 
-    // /event POST
-    /* 
-    Parameters:
-        {
-            "name": string,
-            "description": string,
-            "duration": int
-        }
-    Returns:
-        Success - 200 Status Code
-        {
-            "data": {
-                "id": int
-            }
-        }
-        Error - 400 Status Code
-        {
-            "message": string
-        }
-    */
     router.post('/', async (req, res) => {
         if (!req.body)
             return res.status(400).json({
-                message: "Malformed Request"
+                error: "Malformed Request"
```

Comment:
For this return as well as the error return below, can you also add an empty "data" field? Makes things more consistent if every API return has both data and error, regardless of what it is